infrastructure/base/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "aws_region" {
+  type        = string
+  description = "AWS region"
+  default     = "ap-northeast-2"
+}
+
+variable "allowed_account_id" {
+  type        = string
+  description = "AWS account id"
+}
+
+variable "project" {
+  type        = string
+  description = "Short name of the project, will be used to prefix created resources"
+  default     = "vizz"
```

Comment:
I'd drop all the "vizz" defaults, even if it requires us to explicitly define the values on the tfvars file